backend/pretix/__init__.py

```diff
@@ -347,3 +348,34 @@ def create_order(conference: Conference, order_data: CreateOrderInput) -> Order:
     data = response.json()
 
     return Order(code=data["code"], payment_url=data["payments"][0]["payment_url"])
+
+
+@dataclass
+class Conference:
+    pretix_organizer_id: str
+    pretix_event_id: str
+
+
+def user_has_admission_ticket(
+    *, email: str, event_organizer: str, event_slug: int
```

Comment:
```suggestion
    *, email: str, event_organizer: str, event_slug: str
```
Think the `event_slug` is a string as well